docs/api-security/10-LevelofAssurance.md

```diff
@@ -18,17 +24,33 @@ Level of Assurance (LoA) has been defined to support the OpenID Connect API secu
 
 - Ensure the API consumer has the appropriate authorisation to act on behalf of the end user
 
-API providers **SHOULD** adopt a Level of Assurance model which can be applied when API consumers are accessing health records that are classed as MEDICAL IN-CONFIDENCE.
+API providers **MUST** adopt a Level of Assurance model which can be applied when API consumers are accessing health records that are classed as MEDICAL IN-CONFIDENCE.
+
+API providers **MUST** indicate the LoA associated with a health sector participants authentication and authorisation process in any issued tokens.
+
+### Level of Assurance
+
+Issued tokens can include the following claims that relate to the Level of Assurance (LoA) of the user authentication.
 
-API providers **MUST** indicate the LoA associated with a health sector participants authentication and authorisation process in any issued access tokens.
+The LoA captures the strength of the authentication method used and provides an indication of the trust that the user's identity is correct.
+
+The API Consumer uses these claims to confirm the level of trust that the user is who they say they are and based on this what level of grant.
```

Comment:
```suggestion
The API Provider uses these claims to confirm the level of trust that the user is who they say they are, and based on this, what level of access should be granted.
```

Reply:
Unless I misunderstand the intent, this should be the API Provider?

Reply:
I concur